giskard/ml_worker/websocket/utils.py

```diff
@@ -118,7 +114,7 @@ def log_artifact_local(local_file, artifact_path=None):
     file_name = os.path.basename(local_file)
 
     paths = (projects_dir, artifact_path, file_name) if artifact_path else (projects_dir, file_name)
-    artifact_file = posixpath.join("/", *paths)
+    artifact_file = os.path.join("/", *paths)
```

Comment:
I'm not of this, especially on windows.

Also, projects_dir is a Path already

Could be better to to 

artifact_file  = projects_dir
if artifact_path:
  artifact_file = artifact_file / artifact_path
artifact_file = artifact_file / file_name

Reply:
Done